src/VariableInput/VariableInput.js

```diff
@@ -129,11 +142,20 @@ class VariableInput extends React.PureComponent {
           placeholder={placeholder}
           onBlur={() => setTimeout(this._onBlur, 0)}
           readOnly={disabled}
+          {...oneLineProps}
```

Comment:
What do you think about:
```suggestion
          {!multiline && ...singleLineProps}
```

When:
```js
   const singleLineProps = {
          handlePastedText: this._handlePastedText,
          handleReturn: () => 'handled',
    };
```

I think it's a little more readable